src/plugins/janus_streaming.c

```diff
@@ -466,7 +466,8 @@ multistream-test: {
 	"new_secret" : "<new secret for the mountpoint; optional>",
 	"new_pin" : "<new PIN for the mountpoint, PIN will be removed if set to an empty string; optional>",
 	"new_is_private" : <true|false, depending on whether the mountpoint should be now listable; optional>,
-	"permanent" : <true|false, whether the mountpoint should be saved to configuration file or not; false by default>
+	"permanent" : <true|false, whether the mountpoint should be saved to configuration file or not; false by default>,
+	"edited_event" : <true|false, wheter janus will send an edited event to all viewers when metadata is updated; false by default>
```

Comment:
There's a typo on "whether". And it's not "janus" that sends the event: I'd just generically say

> whether an event will be sent...

Reply:
Typo fixed!